ci-operator/config/openshift/aws-karpenter-provider-aws/openshift-aws-karpenter-provider-aws-main.yaml

```diff
@@ -0,0 +1,41 @@
+base_images:
+  base:
+    name: "4.19"
+    namespace: ocp
+    tag: base
+binary_build_commands: make binary
+build_root:
```

Comment:
Most folks manage their build_root with a file in their repo these days. It is much easier to change it alongside a golang bump in the repository.
See https://docs.ci.openshift.org/docs/architecture/images/#controlling-go-versions-in-component-repositories and `.ci-operator.yaml`. If you have this file in your repository, your ci-operator looks like:
```
build_root:
  from_repository: true
```

Reply:
yeah, that makes good sense. will update.